spec/meilisearch/client/errors_spec.rb

```diff
@@ -1,6 +1,26 @@
 # frozen_string_literal: true
 
 RSpec.describe 'MeiliSearch::Client - Errors' do
+  describe 'MeiliSearch::Error' do
+    it 'catches MeiliSearch::TimeoutError' do
+      expect do
+        raise MeiliSearch::TimeoutError
+      end.to raise_error(MeiliSearch::Error)
```

Comment:
WDYT?
```suggestion
      expect(MeiliSearch::TimeoutError).to be_a MeiliSearch::Error
```

Reply:
Maybe even reducing to a single multi-assertion test.

Reply:
I agree that makes more sense. I got caught trying to "test behavior" instead of doing the obvious thing.

Please see if the current solution is to your liking, another alternative I was considering is looping through `MeiliSearch.constants.select { |c| c.ends_with? 'Error' }` and asserting on each of them in order to catch future error classes (and automatically keep the spec up to date) too but it seemed like an overengineered solution.

Reply:
It was good enough the way you did!